Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM ubuntu:18.04
+
+# Install required packages and remove the apt packages cache when done.
+
+RUN apt-get update && \
+    apt-get upgrade -y && \
+    apt-get install -y \
+        git \
+        python3 \
+        python3-dev \
+        python3-setuptools \
+        python3-pip \
+        nginx \
+        cron \
+        supervisor && \
+        pip3 install -U pip setuptools && \
+    rm -rf /var/lib/apt/lists/*
```

Comment:
Again, any reasons for doing so?

Reply:
I am not very experienced in Linux. But I have seen this in many Dockerfiles.

Reply:
Okay. We'll hardly need to do an `apt update` on a running container.